### ISSUE TITLE
How to realize asynchronous task.

### ISSUE THREAD

**Issue by levylll** [getsentry/raven-python#949]
Imaging a scenario: When I catch exceptions, I don't send the message to sentry immediately, I just remember it in memory in a global list called sentry_list. At the same time I have a process which read the list sentry_list. If there is a record in sentry_list, the process will send it to sentry.
I know I can use captureMessage to send message. But captureMessage can not record enough message as captureException does. 
What do you think of my idea. And how to realize it.

**Comment by dcramer** [comment]
@levylll you cant really do this. Sentry needs access to the runtime generated exception object (of which there is only one active). The only thing you could do is capture all of the information immediately and defer sending (which is what we do by default).

tl;dr you need sys.exc_info(), and then you need to gc it correctly if you're capturing it by hand
